includes/admin/helper/class-wc-helper.php

```diff
@@ -1527,12 +1527,6 @@ private static function _prompt_helper_connect( $screen_id ) {
 
 		$active_woo_plugins = array_intersect_key( $woo_plugins, array_flip( $active_plugins ) );
 
-		if ( count( $active_woo_plugins ) > 0 ) {
```

Comment:
Now this function there's no use, but will keep running in `WC_Helper::admin_notices`.
Also this a private method, so we can just remove it.

Reply:
Nice catch @claudiosanches! It's a good idea!
I removed the function in the commit  [134c0ea](https://github.com/woocommerce/woocommerce/pull/26167/commits/134c0ea0db8fd934053906f7c68beedf5af2ea30)